docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>I'm not sure about the "start adding terms to", since we can't add terms anymore. Now you just call <code>.fit()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>